docs/UserGuide.md

```diff
@@ -171,9 +172,28 @@ Examples:
   <br>
   ![result for 'find John'](images/findJohn.png)
 
+### Copying student emails: `copy`
+
+Copy student emails to the clipboard.
+
+Format: `copy [t/TAG]…`
+
+* When no tags are supplied, copies all student emails.
+* Tag is searched as complete tag names. e.g. `copy t/tutorial-1` will only return those with a tag 'tutorial-1'
+(tag 'tutorial-10', 'tutorial-11', etc. will NOT be copied)
+* Copied emails are in the form of `example1@email.com, example2@email.com, example3@email.com` and adheres to the format specified in [section 3.4 of RFC5322](https://tools.ietf.org/html/rfc5322#section-3.4).
+  * This makes it easy to paste into `Gmail`, `Outlook`, `Yahoo Mail`, etc.
+  <br>
+  <img src="images/copyToGmail.png" alt="result for 'copy t/tutorial-2'" style="border:4px solid #808080"/>
+
+Examples:
+* `copy` copies all emails
+* `copy t/tutorial-2` copies 3 emails (`johnsmith@example.com, irfan@example.com, royb@example.com` all with `tutorial-2` tag)
+* `copy t/tutorial-2 t/strong-student` copies 1 email (`royb@example.com` with both `tutorial-2` and `strong-student` tag) 
+
 ### Deleting a student : `delete`
 
-Deletes a specified student from the address book.
+Deletes a specified student.
```

Comment:
I also noticed that we have some phrases of'address book' in UG/DG, we should really remove them now as EduConnect is not an address book any more